index.bs

```diff
@@ -1149,9 +1149,64 @@ The [=new=] {{MLGraphBuilder}} constructor steps are:
 1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the [=webnn-feature|webnn=] feature, throw a "{{SecurityError}}" {{DOMException}} and abort these steps.
 1. Let |context| be the first argument.
 1. If the <a>validate MLContext</a> steps given |context| return `false`, throw a "{{TypeError}}" and abort these steps.
-
 1. Set {{MLGraphBuilder/[[context]]}} to |context|.
 
+### The constant() method  ### {#api-mlgraphbuilder-constant-method}
+Create a constant {{MLOperand}} that can be used in {{MLGraphBuilder}} methods.
+
+#### The {{MLGraphBuilder/constant(descriptor, bufferView)}} method  #### {#api-mlgraphbuilder-constant}
+<div class="note">
+    **Arguments:**
+        - *descriptor*: an {{MLOperandDescriptor}} object
+        - *bufferView*: an {{MLBufferView}}
+    **Returns:**: an {{MLOperand}} object.
+</div>
+<div algorithm=constant>
+    The {{MLGraphBuilder/constant(descriptor, bufferView)}} steps are:
+    <div class="note">
+        The permissions and context validity have been checked by [[#api-mlgraphbuilder-constructor]] steps.
+    </div>
+    1. Let |descriptor| be the first argument.
+    1. If |descriptor| is not an an object that [=implements=] {{MLOperandDescriptor}}, then throw a "{{TypeError}}" {{DOMException}} and stop.
+        1.  If the [=byte length=] of |descriptor| is not supported by the underlying platform, then throw a "{{DataError}}" {{DOMException}} and stop.
+        1. If the [=check dimensions=] steps given |descriptor|.{{MLOperandDescriptor/type}} and |descriptor|.{{MLOperandDescriptor/dimensions}} return `false`, throw a "{{DataError}}" {{DOMException}} and stop.
+    1. Let |bufferView| be the second argument.
+        1. If |bufferView| is not an {{MLBufferView}}, then throw a "{{TypeError}}" {{DOMException}} and stop.
+        1. If |descriptor|.{{MLOperandDescriptor/type}} is not equal to |bufferView|'s [=element type=], throw a "{{DataError}}" {{DOMException}} and stop.
+        1. If the [=byte length=] of |descriptor| is not equal with |bufferView|.\[[ByteLength]], throw a "{{DataError}}" {{DOMException}} and stop.
+    1. Let |operand| be the result of invoking the <a>create MLOperand</a> steps with [=this=], `"constant"` and |descriptor|.
+        1. If that throws, re-throw the exception and stop.
+    1. Make a request to the underlying platform to register |operand| as a constant with |bufferView| as value.
+        1. If that fails, throw an "{{OperationError}}" {{DOMException}} and stop.
+    1. Return |operand|.
```

Comment:
I suppose there should be an internal slot of `MLOperand` that represents the platform operand object. Otherwise, when this `MLOperand` be passed to an operator build method, e.g. `conv2d(input, filter)` where filter is a constant `MLOperand`, the algorithm of `conv2d` would refer to the corresponding platform operand object of `filter` to create a `conv2d` platform operator object.

Reply:
Cross refer to https://github.com/webmachinelearning/webnn/pull/337/files#r1139520188

Reply:
Under consideration how to do it correctly... I will come up with something today.

Reply:
OK, pushed one version, let's see how do you like it.